index.html

```diff
@@ -4,11 +4,34 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>D&D Companion</title>
+<!-- Bootstrap version 5.3.3 for CSS -->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css">
+
+<!-- Our version of style -->
     <link rel="stylesheet" href="./assets/css/style.css">
+
+<!-- Bootstrap version 5.3.3 for JS -->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/js/bootstrap.bundle.min.js">
```

Comment:
this needs to be a "src" in a script tag in the bottom of the body above our own JS script src on line 22